tests/nfv/run_performance_tests.pm

```diff
@@ -15,16 +15,57 @@ use testapi;
 use strict;
 use lockapi;
 
+our $results_dir = '/tmp';
+our $ovs_version;
+our $test_url;
+
+sub run_test {
+    my ($test, $vswitch) = @_;
+    my $testname = "$test\_$vswitch";
+    record_info($testname);
+    assert_script_run("./vsperf --conf-file=/root/vswitchperf/conf/10_custom.conf --vswitch $vswitch $test", timeout => 4000);
+    record_info("Push Results");
+    assert_script_run(sprintf('./push2db.py --parsefolder "%s" --targeturl http://10.86.0.128:8086 --os_version %s --os_build %s --vswitch_version %s --openqa_url %s',
+            $results_dir, get_var('VERSION'), get_var('BUILD'), $ovs_version, $test_url));
+    assert_script_run("mv /tmp/push2db.log /tmp/push2db_$testname.log");
+    upload_logs("/tmp/push2db_$testname.log", failok => 1);
+    assert_script_run("tar -czvf /tmp/vsperf_logs_$testname.tar.gz $results_dir/results_*");
+    upload_logs("/tmp/vsperf_logs_$test.tar.gz", failok => 1);
+    assert_script_run("rm -r $results_dir/results_*");
```

Comment:
tests meant to be running in BM, right? so each single letter typed via IPMI matters and should have good reasons to do so. This `results_*` folders are unique for each run, right? Let's think about how we can optimize this to do it in more lazy way :)

Reply:
Each test run creates a directory results_<timestamp>. Now push2db is executed after each test. 
push2db takes all these directories that start with the string "results_" and will process each of them and push to the DB the CSV. 
I want to make sure that there is only 1 directory with this format so that we don't push the same results 2 or more times.